installer/Migration/V4_3_3/Migration.php

```diff
@@ -25,12 +25,55 @@
 
 class Migration extends AbstractMigration
 {
+    public const CONFLICTING_FOREIGN_KEY_TABLES = [
+        'hs_hr_emp_history_of_ealier_pos',
```

Comment:
Oops. I have never seen this used. We might have to drop unused tables from the DB. I guess almost all the tables should have an entity class except `migration` table.